### PR TITLE
translatedocker-compos.md document to chinese

### LINKED ISSUE #4906: Add several Chinese version documents
Hi,  there are some files in English version, we can add a Chinese version for each:
- [ ] https://github.com/apache/incubator-shardingsphere/blob/master/examples/docker/docker-compose.md
- [x] #2923 https://github.com/apache/incubator-shardingsphere/blob/master/examples/README.md
- [x] #4909 https://github.com/apache/incubator-shardingsphere/blob/master/sharding-scaling/README.md 
- [x] $4914 https://github.com/apache/incubator-shardingsphere/blob/master/shardingsphere-ui/README.md
- [x] $4914 https://github.com/apache/incubator-shardingsphere/blob/master/shardingsphere-ui/shardingsphere-ui-frontend/README.md

A tip: Chinese version file name should be origin filename with a "_ZH". If origin filename is "README.md", the new filename should be "README_ZH.md".

These tasks are good for new contributors, reply here to pick any task you want.

### PR DESCRIPTION
**Pull request by edidada**
Fixes #4906.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: examples/docker/docker-compose-zh.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Using docker-compose to config startup environment
```
Comment (kimmking) [inline examples/docker/docker-compose-zh.md]: remove english version.

File: examples/docker/docker-compose-zh.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+#### sharding-jdbc
+1. 运行'cd docker/sharding-jdbc/sharding'，进入docker文件夹
+2. 运行'docker-compose up -d'，启动docker compose环境
+3. 根据需要，开启mysql / etcd / zookeeper
+4. 如果有端口冲突，在docker-compose.yml中修改相应的端口，然后再次使用'docker-compose up -d'启动docker compose
+5. 如果需要关闭相关程序环境，使用命令'docker-compose down'
+
+#### sharding-proxy
+1. access the docker folder (cd docker/sharding-proxy/sharding)
```
Comment (kimmking) [inline examples/docker/docker-compose-zh.md]: only chinese here

**Comment by kimmking** [comment]
@edidada we should modify the request changes.

**Comment by edidada** [comment]
> @edidada we should modify the request changes.

@kimmking I have modify it

File: examples/docker/docker-compose-zh.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## 使用docker-compose去初始化开始环境
```
Comment (kimmking) [inline examples/docker/docker-compose-zh.md]: remove 去

File: examples/docker/docker-compose-zh.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+在开始使用docker compose之前，根据下述参考网址安装docker和docker-compose
```
Comment (kimmking) [inline examples/docker/docker-compose-zh.md]: miss url from  the origin：
> before we use docker compose, please install docker and docker-compose first : https://docs.docker.com/compose/install/

File: examples/docker/docker-compose-zh.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+#### sharding-jdbc
+1. 运行'cd docker/sharding-jdbc/sharding'，进入docker文件夹
+2. 运行'docker-compose up -d'，启动docker compose环境
+3. 根据需要，开启mysql / etcd / zookeeper
```
Comment (kimmking) [inline examples/docker/docker-compose-zh.md]: remove whitespaces

File: examples/docker/docker-compose-zh.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+5. 如果需要关闭相关程序环境，使用命令'docker-compose down'
+
+#### sharding-proxy
+1. 运行'cd docker/sharding-jdbc/sharding'，进入docker文件夹
+2. 运行'docker-compose up -d'，启动docker compose环境
+3. 运行 `mysql -h127.0.0.1 -P13308 -proot -uroot`登录代理
+4. 如果有端口冲突，在docker-compose.yml中修改相应的端口，然后再次使用'docker-compose up -d'启动docker compose
+5. 如果需要关闭相关程序环境，使用命令'docker-compose down'
+
+谨慎使用docker删除指令`docker ps -a -q`去删除docker容器
```
Comment (kimmking) [inline examples/docker/docker-compose-zh.md]: it's better 需要注意，请谨慎使用....

**Comment by geomonlin** [comment]
> Fixes #4906.
> 
> ## Changes proposed in this pull request:

Is it convenient to send an email or wx for better communication

**Comment by tristaZero** [comment]
Thanks for your updating. @edidada

**Pull request merged**